src/components/Cancel/Cancel.jsx

```diff
@@ -0,0 +1,14 @@
+import { StyledCancelLight, StyledCancelDark } from "./StyledCancel.styled";
+
+const Cancel = (props) => {
+    switch (props.cancelStyle) {
+        case 'light': 
+            return <StyledCancelLight />;
+        case 'dark': 
+            return <StyledCancelDark />;
+        default: 
+            return  <StyledCancelDark />;
+    }
+}
+
+export default Cancel;
```

Comment:
pls open Button component in dev and look how you can implement this switch case statement 
Like this:
![image](https://github.com/whatislove23/deep-sound/assets/100228936/51b65ad2-e140-425b-844f-d1a3913e0169)

Reply:
No problem. Done!